lib/pages/classes/view/classes_page.dart

```diff
@@ -60,6 +62,16 @@ class _ClassesPageState extends State<ClassesPage> {
       // TODO(IoanaAlexandru): Simply show all classes if user is not authenticated
       needsToBeAuthenticated: true,
       actions: [
+        if (RemoteConfigService.feedbackEnabled)
+          AppScaffoldAction(
+            icon: Icons.format_list_numbered_outlined,
```

Comment:
It's not clear what this icon means in this context. Maybe just use the feedback icon here as well.